CrazyCanvas/Source/ECS/Systems/Match/ClientFlagSystem.cpp

```diff
@@ -125,22 +125,6 @@ void ClientFlagSystem::TickInternal(LambdaEngine::Timestamp deltaTime)
 	{
 		Entity flagEntity = m_Flags[0];
 
-		PacketComponent<FlagEditedPacket>& flagPacketComponent = pECS->GetComponent<PacketComponent<FlagEditedPacket>>(flagEntity);
-		const TArray<FlagEditedPacket>& flagEditedPackets = flagPacketComponent.GetPacketsReceived();
-
-		for (const FlagEditedPacket& editedPacket : flagEditedPackets)
-		{
-			switch (editedPacket.FlagPacketType)
-			{
-			case EFlagPacketType::FLAG_PACKET_TYPE_PICKED_UP:
-				OnFlagPickedUp(MultiplayerUtils::GetEntity(editedPacket.PickedUpNetworkUID), flagEntity);
-				break;
-			case EFlagPacketType::FLAG_PACKET_TYPE_DROPPED:
-				OnFlagDropped(flagEntity, editedPacket.DroppedPosition);
-				break;
-			}
-		}
-
 		const ParentComponent& parentComponent = pECS->GetConstComponent<ParentComponent>(flagEntity);
 
 		if (parentComponent.Attached)
```

Comment:
should not a const component call `pECS->GetConstComponent<xxxComponent>` instead of `pECS->GetComponent<xxxComponent>` ?

Reply:
(in the lines under 146)